packages/expo/package.json

```diff
@@ -55,6 +55,7 @@
   },
   "homepage": "https://github.com/expo/expo/tree/master/packages/expo",
   "dependencies": {
+    "@approvals-cloud/badgin": "^1.1.0",
```

Comment:
Would you care to change it to `badgin`?

<img width="401" alt="Zrzut ekranu 2019-11-20 o 17 31 51" src="https://user-images.githubusercontent.com/1151041/69257616-a4ce6100-0bbb-11ea-9178-709916b9e34d.png">